src/components/tasks/KanbanBoard.tsx

```
@@ -0,0 +1,163 @@
+                tasks: sourceTasks
+            },
+            [destColId]: {
+                ...endCol,
+                tasks: destTasks
+            }
+        };
+
+        setColumns(newColumns); // Optimistic update
+
+        // --- Database Updates ---
+        const dbUpdates: Promise<any>[] = [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)